Introduction to Kotlin Multiplatform/05_reuse_code.md

```diff
@@ -36,11 +36,11 @@ const val jvmPort = 8888
 The declarations from the Kotlin files under the
```

Comment:
The declarations from the Kotlin files under
`src/commonMain/kotlin` are visible to all targets. We can only
use the platform-independent subset of the Kotlin standard library
from the Kotlin/Common source files. The Kotlin Multiplatform libraries are allowed too.
It allows us to easily share code between platforms.